sandbox/OpenIddict.Sandbox.Avalonia.Client/OpenIddict.Sandbox.Avalonia.Client/ViewModels/MainViewModel.cs

```diff
@@ -0,0 +1,194 @@
+﻿using Avalonia.Controls;
+using OpenIddict.Abstractions;
+using OpenIddict.Client;
+using ReactiveUI;
+using System;
+using System.Collections.Generic;
+using System.Reactive;
+using System.Reactive.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using static OpenIddict.Abstractions.OpenIddictConstants;
+using static OpenIddict.Abstractions.OpenIddictExceptions;
+
+namespace OpenIddict.Sandbox.Avalonia.Client.ViewModels;
+
+public class MainViewModel : ViewModelBase
+{
+#pragma warning disable CA1822 // Mark members as static
+    public string Greeting => "Welcome to Avalonia!";
+
+    private OpenIddictClientService _service;
+
+    public ReactiveCommand<Unit,Unit> LoginCommand { get; }
+    public ReactiveCommand<Unit, Unit> LoginWithGithubCommand { get; }
+    public ReactiveCommand<Unit, Unit> LogoutCommand { get; }
+    public ReactiveCommand<Unit, Unit> CancelCommand { get; }
+#pragma warning restore CA1822 // Mark members as static
+
+    public MainViewModel(OpenIddictClientService service)
+	{
```

Comment:
Formatting seems off there too.

Reply:
Ok, I changed my vs settings from tabs to spaces and re-formatted